builds/misc/addons-publish.yaml

```diff
@@ -23,42 +23,17 @@ jobs:
             clean: true
             fetchDepth: 100
 
-          # Both docker logins needed for if we need to test this job. In this case images should go to edgebuilds.
           - task: Docker@2
-            displayName: Docker login edgebuilds
+            displayName: Docker login
             inputs:
               command: login
-              containerRegistry: iotedge-edgebuilds-acr
-
-          - task: Docker@2
-            displayName: Docker login edgerelease
-            inputs:
-              command: login
-              containerRegistry: iotedge-release-acr
+              containerRegistry: $(service-connection.registry)
```

Comment:
Sorry, not sure what this means?

Reply:
It still exists in DevOps has a registered service connection. And it will be added as `service-connection.registry` to the pipeline's variables in the UI as a separate step right after this gets merged.